lexicographical_sorter/model/LexicographicalSorter.php

```diff
@@ -0,0 +1,35 @@
+<?php
+namespace Sorter;
+require_once __DIR__ . '/Sorter.php';
+require_once __DIR__ . '/../ui/Items.php';
+use UI\Items;
+
+class LexicographicalSorter implements Sorter
+{
+    private $output;
```

Comment:
this class is stateful but there's no need for it, it could return the output without storing it in a field and simplify itself